app/index.html

```diff
@@ -8,11 +8,27 @@
 
     <link rel="manifest" href="/manifest.json">
     <link rel="icon" href="/favicon.ico">
+    <style>
+      .splash-screen {
+        display: flex;
+        align-items: center;
+        justify-content: center;
+        height: 100vh;
+      }
+
+      .splash-screen > div {
+        max-width: 50%;
+      }
+    </style>
     <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:regular,bold,italic,thin,light,bolditalic,black,medium&amp;lang=en">
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
   </head>
   <body>
-    <div id="root"></div>
+    <div id="root">
+      <div class="splash-screen">
```

Comment:
This could have been also solved on the react side.. not sure what is better...
Have `Loading` component that would be displayed as long as `state.isInitializing == true`. Then we can init even more things than just auth (if needed)